system/handlers/admin/AdHocTaskManager.cfc

```diff
@@ -44,15 +45,21 @@ component extends="preside.system.base.AdminHandler" {
 		var taskTitleData = IsJson( prc.task.title_data ?: "" ) ? DeserializeJson( prc.task.title_data ) : [];
 		var taskTitle = translateResource( uri=prc.task.title, data=taskTitleData, defaultValue=prc.task.title );
 
-		prc.taskProgress           = adHocTaskManagerService.getProgress( taskId );
+		prc.taskProgress = adHocTaskManagerService.getProgress( taskId );
 
 		if ( prc.task.status == "pending" ) {
 			prc.taskProgress.timeTaken = translateResource( "enum.adhocTaskStatus:pending.title" );
-			prc.taskProgress.log       = taskManagerService.createLogHtml( translateResource( "cms:adhoctaskmanager.progress.pending.log" ) );
+			prc.taskProgress.log       = logRendererUtil.renderLegacyLogs( translateResource( "cms:adhoctaskmanager.progress.pending.log" ) );
+			prc.taskProgress.lineCount = 0;
 		} else {
+			if ( Len( prc.taskProgress.log ) ) {
+				prc.taskProgress.lineCount = ListLen( prc.taskProgress.log, Chr( 10 ) );
+				prc.taskProgress.log = logRendererUtil.renderLegacyLogs( prc.taskProgress.log );
+			} else {
+				prc.taskProgress.log = logRendererUtil.renderLogs( adhocTaskManagerService.getLogLines( prc.taskProgress.id ) );
```

Comment:
For clarity can we switch these round so they are .lineCount and then .log in the same order as the lines above?